### PR TITLE
refactor for internal serving build

### PR DESCRIPTION
**Pull request by Orion34-lanbo**

### CODE REVIEW

File: scripts/python/common_setup.py [hunk 1]
```diff
@@ -619,9 +619,9 @@ def symlink_disc_files(args):
         )
     )
 
-    logger.info("linking PatineClient")
-    link_dirs(os.path.join(dir_tf_community, 'tao', 'third_party', 'PatineClient'),
-            os.path.join(dir_platform_alibaba, 'third_party', 'PatineClient'))
+    # logger.info("linking PatineClient")
+    # link_dirs(os.path.join(dir_tf_community, 'tao', 'third_party', 'PatineClient'),
+    #         os.path.join(dir_platform_alibaba, 'third_party', 'PatineClient'))
```
Comment (minminsun) [inline scripts/python/common_setup.py:624]: Remove?

Comment (qiuxiafei) [reply scripts/python/common_setup.py:624]: +1

**Review by qiuxiafei — APPROVED** [review]
LGTM. Pls clear the comments ~

File: scripts/python/common_setup.py (hunk 1 above)
Comment (Orion34-lanbo) [reply scripts/python/common_setup.py]: done

**Review by qiuxiafei — APPROVED** [review]

**Review by Yancey1989 — APPROVED** [review]
LGTM

**Pull request merged**